mm2src/mm2_bitcoin/spv_validation/Cargo.toml

```diff
@@ -4,8 +4,14 @@ version = "0.1.0"
 authors = ["Roman Sztergbaum <rmscastle@gmail.com>"]
 
 [dependencies]
-bitcoin-spv = "5.0.0"
 chain = {path = "../chain"}
 primitives = { path = "../primitives" }
+ripemd160 = "0.8.0"
```

Comment:
should update `ripemd160` to "0.9" after merging with https://github.com/KomodoPlatform/atomicDEX-API/pull/1294